src/crypter.js

```diff
@@ -55,6 +59,7 @@ module.exports.Crypter = (secret: Buffer, {
 
       const iv = crypto.randomBytes(ivLength);
       // eslint-disable-next-line no-bitwise
+      // you're basically packing the gzip boolean into the first bit here?
```

Comment:
That's correct. So true randomness of the IV is 127 bits instead of 128. But looking at the change in chance of collision probability this seems acceptable. However I agree with potentially moving this out and keeping this object very simple.